internal/services/appservice/registration.go

```diff
@@ -28,6 +28,7 @@ func (r Registration) DataSources() []sdk.DataSource {
 			LinuxWebAppDataSource{},
 			ServicePlanDataSource{},
 			WindowsWebAppDataSource{},
+			LinuxFunctionAppDataSource{},
```

Comment:
Can we move this up so the list is alphabetical? (Guessing you tried `tfpdk`? I have an open issue to insert this in the right place rather than just append...)